app/pages/new-dose-medication/new-dose-medication.ts

```diff
@@ -17,10 +20,12 @@ export class NewDoseMedicationPage implements OnInit {
     public title: string = "New Dose Medication";
     public doseSeries: DoseSeries;
     public everyday: boolean;
-    public doseTime: Date;
+    public doseTime: string;
```

Comment:
The ionic date picker outputs the Date as a string in the ISO 8601 Datetime Format (YYYY-MM-DDTHH:mmZ).

For this reason, I converted this to a string, and have added the moment.js library to our project, which can easily convert "17:00" into the number of ms since the epoch for today at 5:00 PM.